keyboards/framework/laptop_16/ansi/white/config.h

```diff
@@ -0,0 +1,7 @@
+// Copyright 2022-2023 Framework Computer
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define MATRIX_COLS 16
+#define MATRIX_ROWS 8
```

Comment:
Is this file's location implying future models will have different numbers of rows/cols? If that is the case, I would presume the `matrix.c` code would need modification, meaning this configuration should either be moved upward, or the custom matrix scanning code be moved to the specific keyboard revision.